docs/modules/device-lab-management/pages/index.adoc

```diff
@@ -0,0 +1,6 @@
+= Device lab management
+:navtitle: Device lab management
+
```

Comment:
Can you provide one or two sentence to briefly explain what Device Lab Management is? You can refer to this old doc: https://support.kobiton.com/hc/en-us/categories/360005230692-Device-Lab-Management

Reply:
Wasn't able to find anything solid as a whole about Device Lab Management. But still put together a paragraph based on the documents that are in this section.


"In this section, you will learn about what tools and services Kobiton has and how you can use and manage them. You will learn about:

- Hardware requirement of deviceConnect, which necessitates an M1 or M2 Mac mini.
- Restarting deviceConnect services to apply configuration changes or software updates.
- Retrieving deviceConnect logs for troubleshooting and submitting tickets to Kobiton Support."